docs/reference/commandline/service_create.md

```diff
@@ -761,6 +762,18 @@ $ docker service create --name dns-cache -p 53:53/tcp -p 53:53/udp dns-cache
 $ docker service create --name dns-cache -p 53:53/udp dns-cache
 ```
 
+### Provide credential specs for managed service accounts (Windows only)
+
+This option is only used for services using Windows containers. The
+`--credential-spec` must be in the format `file://spec.txt` or
+`registry://keyname`.
+
+When using the `file://` format, the referenced file must be present on the
+daemon host in the `C:\programfiles\docker\credentialspecs\` directory. When
```

Comment:
Actually it is `$env:ProgramData`, not `$env:ProgramFiles` (PowerShell environment variable syntax), so the default is `$env:ProgramData\Docker\CredentialSpecs` which is usually `C:\ProgramData\Docker\CredentialSpecs`.

Is there a better way to phrase this to say it's `CredentialSpecs` under the Docker data root directory, which defaults to `$env:ProgramData\Docker` where `$env:ProgramData` is usually `C:\ProgramData`?